spec.yaml

```diff
@@ -109,6 +109,51 @@ paths:
             $ref: "definitions.yaml#/definitions/V4GenericResponse"
 
   /v4/clusters/:
+    get:
+      operationId: getClusters
+      tags:
+        - clusters
+      summary: Get clusters
+      description: |
+        This operation allows to fetch a list of clusters.
```

Comment:
@rossf7 As you reviewed already, but didn't comment here: Do you see a problem with the phrase `This operation allows to fetch a list of clusters.` ?